### ISSUE TITLE
Question about reproducing sota results

### ISSUE THREAD

**Issue by betteryy** [RecklessRonan/glognn#2]
Hi, I read the paper and very interested in this method, especially results of Cornell and Chameleon dataset. 

And i have a question when i studying your code..

When i reproduce your results i need your help about hyperparameters. 

This is results of my computer (i referenced the file 'run_glognn++_sota_reproduce_small.sh')

.....
cornell_lr0.01_do0.5_es200_wd0.0001_alpha1.0_beta0.1_gamma0.7_delta1.0_nlid1_nl2_ordersid2_orders2_split0_results.txt
tmp_orders torch.Size([183, 5])
orders_weight torch.Size([1])
tmp_orders torch.Size([183, 5])
orders_weight torch.Size([1])
Test set results: loss= 0.9041 accuracy= 0.6486

The accuracy is significantly different from the paper. Is it some error?  
( In 'run_glognn++_sota_reproduce_small.sh' , python3 main.py --no-cuda --model mlp_norm --epochs 2000 --hidden 64 --lr 0.01 --dropout 0.0 --early_stopping 200 --weight_decay 0.0001 --alpha 1.0 --beta 0.1 --gamma 0.7 --delta 1.0 --norm_layers 2 --orders 2 --orders_func_id 2 --norm_func_id 2 --dataset cornell --split 0)

**Comment by RecklessRonan** [comment]
Hi~
Our results in the paper are calculated by the average of the 10 splits, and you can get the results of the other splits(1-9) by setting the args '--split'.
